stable/traefik/README.md

```diff
@@ -95,6 +95,8 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `memoryRequest`                 | Initial share of memory requested per Traefik pod                    | `20Mi`                                    |
 | `cpuLimit`                      | CPU limit per Traefik pod                                            | `200m`                                    |
 | `memoryLimit`                   | Memory limit per Traefik pod                                         | `30Mi`                                    |
+| `nodeSelector`                  | node labels for pod assignment                            | {}                                    |
```

Comment:
For consistency, capitalize the first word in the description. Also use backticks around the empty map.